tests/test_ops/test_deform_conv.py

```diff
@@ -147,6 +182,12 @@ def test_deformconv(self):
         self._test_deformconv(torch.double)
         self._test_deformconv(torch.float)
         self._test_deformconv(torch.half, threshold=1e-1)
+        # test batch_size < im2col_step
+        self._test_deformconv(torch.float, batch_size=1, im2col_step=2)
+        # test bach_size % im2col_step != 0
+        with pytest.raises(
+                AssertionError, match='im2col step must divide batchsize'):
```

Comment:
May also fix the error message like: `im2col_step must be divisible by batch size